src/validateData.js

```diff
@@ -0,0 +1,45 @@
+const { convertToCase } = require('./convertToCase');
+
+function validateData(reqURL) {
+  const errors = [];
+
+  const [originalText, query] = reqURL.slice(1).split('?');
+  const params = new URLSearchParams(query);
+  const targetCase = params.get('toCase');
```

Comment:
Your function shouldn't know anything about URL and params, it should just get and validate data